common/wonders/wc_wonders_historical.txt

```diff
@@ -247,3 +247,403 @@ wonder_wyrmrest_temple = {
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.1
+			monthly_character_piety = 0.2
+		}
+	}
+	stage = {
+		allow = {
+		}
+		build_time = 325
+		gold_cost_flat = 2600
+		gold_cost_ticking = 5
+		restore_time = 200
+		loot_time = 1
+		restore_cost_flat = 500
+		restore_cost_ticking = 2
+		upgrade_slots = 2
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.2
+			monthly_character_piety = 0.2
+		}
+	}
+}
+# Based on wonder_cathedral
+# Shrine of Storms
+wonder_shrine_storm = {
+	allow_construction = no
+	picture = GFX_wonder_strip_shrine_storm
+	model = wonder_temple
+	potential = {
+		OR = {
+			religion = old_gods_worship
+			religion = tidemother
+		}
+	}
+	active = {
+		ROOT = {
+			OR = {
+				religion = old_gods_worship
+				religion = tidemother
+			}
+		}
+	}
+	flags = {
+		holy_building
+		inspires_piety
+		is_historical
+		cost_medium
+		gives_artifact
+		gives_artifact_bells
+	}
+	available_upgrades = {
+		# VERY UNIQUE:
+		upgrade_crucible_of_storms
+		
+		# UNIQUE:
+		upgrade_gargoyles
+		upgrade_bells
+		upgrade_stained_glass_window
+		upgrade_relic_tomb
+		upgrade_grand_basilica
+		upgrade_catacombs
+		upgrade_giant_cross
+		upgrade_bells_golden
+
+		#
+		upgrade_torture_chamber
+		upgrade_historical_mural
+
+		# OTHER:
+	 	upgrade_roads
+		upgrade_watchtower_network
+		upgrade_spikes
+		upgrade_tower
+		upgrade_hypocausts
+		upgrade_heating_pipes
+		# upgrade_retinue # Replaced by upgrade_crucible_of_storms
+		upgrade_road_tolls
+		upgrade_tavern
+	}
+	stage = {
+		allow = {
+		}
+		build_time = 325
+		gold_cost_flat = 2000
+		gold_cost_ticking = 3
+		restore_time = 100
+		loot_time = 6
+		restore_cost_flat = 400
+		restore_cost_ticking = 2
+		upgrade_slots = 2
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.05
+			monthly_character_piety = 0.1
+		}
+	}
+	stage = {
+		allow = {
+		}
+		build_time = 375
+		gold_cost_flat = 2200
+		gold_cost_ticking = 5
+		restore_time = 200
+		loot_time = 6
+		restore_cost_flat = 500
+		restore_cost_ticking = 2
+		upgrade_slots = 2
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.1
+			monthly_character_piety = 0.1
+		}
+	}
+	stage = {
+		allow = {
+		}
+		build_time = 375
+		gold_cost_flat = 2400
+		gold_cost_ticking = 5
+		restore_time = 200
+		loot_time = 6
+		restore_cost_flat = 500
+		restore_cost_ticking = 2
+		upgrade_slots = 2
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.1
+			monthly_character_piety = 0.2
+		}
+	}
+	stage = {
+		allow = {
+		}
+		build_time = 325
+		gold_cost_flat = 2600
+		gold_cost_ticking = 5
+		restore_time = 200
+		loot_time = 1
+		restore_cost_flat = 500
+		restore_cost_ticking = 2
+		upgrade_slots = 2
+		local_modifier = {
+		}
+		owner_modifier = {
+			monthly_character_prestige = 0.2
+			monthly_character_piety = 0.2
+		}
+	}
+}
+# Ulduar
+# Based on wonder_fortress
+wonder_ulduar = {
```

Comment:
I don't know why, but Ulduar gives bonuses even when not active.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77257403-78bc0d00-6c8d-11ea-97cd-e810ebbc6a5d.png)

</details>

The Sunwell from the same file doesn't do it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/77257422-9d17e980-6c8d-11ea-9502-b20ec9004ae0.png)

</details>

Reply:
no idea why. will try to find out whats wrong